tools/compileProtos.ts

```diff
@@ -314,19 +317,24 @@ export async function generateRootName(directories: string[]): Promise<string> {
  */
 export async function main(directories: string[]): Promise<void> {
```

Comment:
nit: The parameter name `directories` confuses me at first glance. If we could change to arguments or other better name, it would great to read, since we add options `--skip-json`

Reply:
Agreed. In the perfect world we should use `yargs` here, but I'm too lazy at this moment :)  Will change to `arguments`.

Reply:
Or rather to `parameters`, since `arguments` is reserved.